README.md

```diff
@@ -1,39 +1,52 @@
-Hey there, it looks like you want to setup a Vagrant VM to develop [Pebble](https://getpebble.com/) watch faces and apps on.
+# vagrant-pebble
+> Vagrant Setup for Pebble SDK
 
-This is the right place to look!
+This Vagrant configuration will configure a VM with all the required dependencies for
+Pebble app development closely following this [guide](http://developer.getpebble.com/sdk/install/linux/).
 
-Once you're done, you should be able to run:
+## Prerequisites
 
-```
-$ vagrant up
-$ vagrant ssh
-vagrant@vagrant-ubuntu-trusty-64:~$ pebble --version
-# PebbleSDK 2.9
-```
+- [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+  - _tested with version 4.3_
+- [Vagrant](http://www.vagrantup.com/downloads)
+  - _tested with version 1.7.2_
 
-Yep, that's right. [Pebble SDK](https://developer.getpebble.com/) v2.9 in a Vagrant image.
+## Getting Started
 
-It's not pretty, so pull requests are happily taken.
+Clone this repository
 
-Use it at your own risk.
+    git clone https://github.com/shalecraig/vagrant-pebble
+    cd vagrant-pebble
 
-Instructions
-------------
+_Optional:_ Checkout the tag matching desired SDK version, `master` is on v3.0-dp7.
```

Comment:
@shalecraig I guess some devs would like to develop against v2.9 instead of v3.0 since it still under dp?